### ISSUE TITLE
bsd:u IClient: 31 (EventFd) is not implemented

### ISSUE THREAD

**Issue by EmulationFanatic** [Ryujinx/Ryujinx#1440]
bsd:u IClient: 31 (EventFd) is not implemented

[Switchbrew](https://switchbrew.org/wiki/Sockets_services#bsd:u.2C_bsd:s) | 

## Service description:
```
All the services commands but the first two return -1 on failure and set errno when that happens. Although Nintendo has the FreeBSD kernel's to socket stack, the errno macro definitions being in use actually come from Linux (and not from FreeBSD as one would expect!).

These services have max_sessions 0x40. There's 22 IPC handler threads for bsd:u, and 11 for bsd:s.
```

## Required by:

https://github.com/Ryujinx/Ryujinx-Games-List/issues/2184
https://github.com/Ryujinx/Ryujinx-Games-List/issues/2966

**Comment by LukeWarnut** [comment]
Required by Ryujinx/Ryujinx-Games-List#3111 when launching Freeplay.

**Comment by AcK77** [comment]
Fixed in #2960
